app/components/client/devprogram-scripts.js

```diff
@@ -0,0 +1,37 @@
+"use strict"; /* global document, history, send, window */
+
+
+
+document.getElementById("get-started").onclick = event => {
+  event.preventDefault();
+
+  send({
+    message: "auth me with github"
+  });
+};
+
+if (window.location.search.includes("?code=")) {
+  document.querySelector("developer-program").innerHTML = `
+    <form>
+      <input-submit>
+        <input id="walletAddress" placeholder="Your LBRY wallet address" type="text"/>
```

Comment:
Can this look stylistically similar or identical (and use the same classes/elements) to the input + button on the playground?

Reply:
Do you mean having `lbry` inside the input? Otherwise, the input/button on Playground is going to match what's on this page (not the other way around). It's the new component.

Reply:
OK, if they're the same, consider this resolved. I thought they looked different when I checked, will confirm when PR is ready for review again.